db/migrate/20200919174716_devise_create_users.rb

```diff
@@ -41,4 +39,4 @@ def change
     # add_index :users, :confirmation_token,   unique: true
     # add_index :users, :unlock_token,         unique: true
   end
-end
+end
```

Comment:
Milica and I changed this quite with the email confimation, ect. Is there anything specific that we should know about before we do the merges?

Reply:
Everything should be fine since we didn't change anything. As long as nothing was taken out, I think your merge should be ok.